wobble/__init__.py

```
@@ -25,6 +26,22 @@ def log_calls_decorator(*args, **kwargs):
     return log_calls_decorator
 
 
+def api_error_catcher(fn):
+    def catch_api_error(*args, **kwargs):
+        try:
+            return fn(*args, **kwargs)
+        except jsonrpclib.jsonrpc.ProtocolError:
+            try:
+                method = json.loads(jsonrpclib.history.request)['method']
+                error_message = 'Wobble-API does not include Method "' + method + '"'
```

Is this garanteed to be a method not found error?
We should only catch method not found errors and raise a UnknownApiMethodException , or something like that.
All other jsonrpc exceptions should propage normally. Otherwise we loose distringuishability.

Anyway. Good idea!